atom/browser/native_window_mac.mm

```diff
@@ -754,6 +757,16 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
     [window_ setOpaque:NO];
   }
 
+  if (base::mac::IsAtLeastOS10_12()) {
+    // Create a tab only if tabbing identifier is specified and window has
+    // a native title bar.
+    if (tabbing_identifier_.empty() || transparent() || !has_frame()) {
+      [window_ setTabbingMode:NSWindowTabbingModeDisallowed];
```

Comment:
This was added in 10.12 and so this does not compile against the 10.10 SDK which Electron builds against.

Will probably need forward declarations and some `respondsToSelector` checks.

https://developer.apple.com/reference/appkit/nswindow/1644729-tabbingmode?language=objc

https://travis-ci.org/electron/electron/jobs/216322601

```
../../atom/browser/native_window_mac.mm:764:16: error: instance method '-setTabbingMode:' not found (return type defaults to 'id') [-Werror,-Wobjc-method-access]
      [window_ setTabbingMode:NSWindowTabbingModeDisallowed];
               ^~~~~~~~~~~~~~
../../atom/browser/native_window_mac.mm:339:12: note: receiver is instance of class declared here
@interface AtomNSWindow : EventDispatchingWindow<QLPreviewPanelDataSource, QLPreviewPanelDelegate, NSTouchBarDelegate> {
           ^
../../atom/browser/native_window_mac.mm:764:31: error: use of undeclared identifier 'NSWindowTabbingModeDisallowed'
      [window_ setTabbingMode:NSWindowTabbingModeDisallowed];
                              ^
../../atom/browser/native_window_mac.mm:766:16: error: instance method '-setTabbingIdentifier:' not found (return type defaults to 'id') [-Werror,-Wobjc-method-access]
      [window_ setTabbingIdentifier:base::SysUTF8ToNSString(tabbing_identifier_)];
```